<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QServerTab</class>
 <widget class="QWidget" name="QServerTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>904</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="defaultServerSet">
   <property name="geometry">
    <rect>
     <x>786</x>
     <y>508</y>
     <width>118</width>
     <height>42</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
font: 16px &quot;Microsoft YaHei&quot;;
color:#FFFFFF;
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #45505C,stop: 1.0 #3A4756);
border:none;
}

QPushButton:hover
{
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 rgba(69, 80, 92, 0.9),stop: 1.0 rgba(58, 71, 86, 0.9));
}

QPushButton:pressed
{
background-color:#131516;
}

QPushButton:disabled
{
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 rgba(69, 80, 92, 0.5),stop: 1.0 rgba(58, 71, 86, 0.5));
color: rgb(117, 117, 117);
}</string>
   </property>
   <property name="text">
    <string>设为默认</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="ServerListTab">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>36</y>
     <width>904</width>
     <height>462</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>904</width>
     <height>350</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>904</width>
     <height>496</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget{
color:#B4C4D9;
font: normal 16px Microsoft YaHei;
background-color:#14191C;
border:none;
}
QHeaderView::section{
background-color:rgba(255,255,255,0.1);
color:#B4C4D9;
font: normal 18px Microsoft YaHei;
border:1px solid #14191C;
height:44px;
}

QScrollBar:vertical
{
background: #2A3137;
width:8px; 
border:1px solid #303840;
border-radius:4px;
}
QScrollBar::handle:vertical
{
background:rgba(178,184,190,0.5);;
width:8px; 
border:1px solid #303840;
border-radius:4px;
}
QScrollBar::handle:vertical:hover{
background-color: #333b42;}

QScrollBar::handle:vertical:pressed{
background-color: #131516;}

QScrollBar::add-line:vertical{
background-color: transparent; }

QScrollBar::sub-line:vertical{
background-color: transparent; }
			
QScrollBar::add-page:vertical{
background-color:#1f2327;}
			
QScrollBar::sub-page:vertical{
background-color:#1f2327;}

QScrollBar:horizontal
{
background: #2A3137;
height:8px; 
border:1px solid #303840;
border-radius:4px;
}
QScrollBar::handle:horizontal
{
background:rgba(178,184,190,0.5);;
height:8px; 
border:1px solid #303840;
border-radius:4px;
}
QScrollBar::handle:horizontal:hover{
background-color: #333b42;}

QScrollBar::handle:horizontal:pressed{
background-color: #131516;}

QScrollBar::add-line:horizontal{
background-color: transparent; }

QScrollBar::sub-line:horizontal{
background-color: transparent; }
			
QScrollBar::add-page:horizontal{
background-color:#1f2327;}
			
QScrollBar::sub-page:horizontal{
background-color:#1f2327;}

QTableWidget::item
{   

}
QTableWidget::item::hover
{
background:transparent;
}
QTableWidget::item:selected
{   
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #3C8CE3,stop: 1.0 #206CC3);
color: #FFFFFF;
}

QMenu{
background-color:#1a1f22;
border-top: 0px solid #A9A9A9;
border-bottom:0px solid #A9A9A9;
}
QMenu::item
{
font-size: 14px;
color: #e1e8f1;
border-bottom:1px solid #4a525a;
background-color:transparent;
padding:10px 10px;
}
QMenu::item:selected
{
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #2fafd4,stop: 1.0 #0796df);
}</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustIgnored</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>120</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>25</number>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>名称</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>AE Title</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>IP</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>端口号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>超时设定 (s)</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="ServerConnectTest">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>508</y>
     <width>118</width>
     <height>42</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
font: 16px &quot;Microsoft YaHei&quot;;
color:#FFFFFF;
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #45505C,stop: 1.0 #3A4756);
border:none;
}

QPushButton:hover
{
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 rgba(69, 80, 92, 0.9),stop: 1.0 rgba(58, 71, 86, 0.9));
}

QPushButton:pressed
{
background-color:#131516;
}

QPushButton:disabled
{
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 rgba(69, 80, 92, 0.5),stop: 1.0 rgba(58, 71, 86, 0.5));
color: rgb(117, 117, 117);
}</string>
   </property>
   <property name="text">
    <string>测试连接</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ServerEdit">
   <property name="geometry">
    <rect>
     <x>658</x>
     <y>508</y>
     <width>118</width>
     <height>42</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
font: 16px &quot;Microsoft YaHei&quot;;
color:#FFFFFF;
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #45505C,stop: 1.0 #3A4756);
border:none;
}

QPushButton:hover
{
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 rgba(69, 80, 92, 0.9),stop: 1.0 rgba(58, 71, 86, 0.9));
}

QPushButton:pressed
{
background-color:#131516;
}

QPushButton:disabled
{
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 rgba(69, 80, 92, 0.5),stop: 1.0 rgba(58, 71, 86, 0.5));
color: rgb(117, 117, 117);
}</string>
   </property>
   <property name="text">
    <string>编辑</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ServerAdd">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>508</y>
     <width>118</width>
     <height>42</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
font: 16px &quot;Microsoft YaHei&quot;;
color:#FFFFFF;
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #45505C,stop: 1.0 #3A4756);
border:none;
}

QPushButton:hover
{
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 rgba(69, 80, 92, 0.9),stop: 1.0 rgba(58, 71, 86, 0.9));
}

QPushButton:pressed
{
background-color:#131516;
}

QPushButton:disabled
{
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 rgba(69, 80, 92, 0.5),stop: 1.0 rgba(58, 71, 86, 0.5));
color: rgb(117, 117, 117);
}</string>
   </property>
   <property name="text">
    <string>增加</string>
   </property>
  </widget>
  <widget class="QLabel" name="ServerPageTitle">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>worklist</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ServerDelete">
   <property name="geometry">
    <rect>
     <x>402</x>
     <y>508</y>
     <width>118</width>
     <height>42</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
font: 16px &quot;Microsoft YaHei&quot;;
color:#FFFFFF;
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #45505C,stop: 1.0 #3A4756);
border:none;
}

QPushButton:hover
{
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 rgba(69, 80, 92, 0.9),stop: 1.0 rgba(58, 71, 86, 0.9));
}

QPushButton:pressed
{
background-color:#131516;
}

QPushButton:disabled
{
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 rgba(69, 80, 92, 0.5),stop: 1.0 rgba(58, 71, 86, 0.5));
color: rgb(117, 117, 117);
}</string>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
